app/src/main/res/values/strings.xml

```diff
@@ -279,10 +279,8 @@
     <string name="onboarding_home_enable_notifications_title" tools:ignore="UnusedResources">Notifications help you do more with Firefox</string>
     <!-- Enable notification pre permission dialog description with rationale -->
     <string name="onboarding_home_enable_notifications_description" tools:ignore="UnusedResources">Sync your tabs between devices, manage downloads, get tips about making the most of Firefox’s privacy protection, and more.</string>
-    <!-- Text for the button to enable app notifications on the device -->
-    <string name="onboarding_home_enable_notifications_positive_button" tools:ignore="UnusedResources">Enable notifications</string>
-    <!-- Text for the button to not enable app notifications on the device and dismiss the dialog -->
-    <string name="onboarding_home_enable_notifications_negative_button" tools:ignore="UnusedResources">No thanks</string>
```

Comment:
these need the `moz:removedIn="XXX"` treatment.

Reply:
That's a good point.  However, there's a change in the requirement again.  I'll be putting these two strings back.

Reply:
what's the sitch on these two strings? Are they out?

Reply:
I landed these strings just yesterday.  I'm not sure if I need to add the _2 if they have not been translated and used do I?